tests/typeclass-tests.dx

```diff
@@ -152,7 +152,17 @@ def f8 {n} [Ix n, Ord' (n=>Int)] (x : n=>Int) : Int = eq x
 
 -------------------- Multi-parameter interfaces --------------------
 
--- TODO!
+-- From PR 1039
+interface Dist d a
+  log_pdf : d -> a -> Float     -- either the density function or mass function
+  draw : d -> Key -> a          -- function for random draws
+
+interface [Dist d a] OrderedDist d a
+  log_cdf : d -> a -> Float     -- the cumulative distribution function (CDF)
+  log_sf : d -> a -> Float      -- the survivor function (complement of CDF)
+  quantile : d -> Float -> a    -- the quantile function (inverse CDF)
```

Comment:
It might be nicer to check in a smaller example that illustrates the same issue, but is not this large.

Reply:
Done.